### ISSUE TITLE
Leverage @wordpress/scripts as our primary tool chain

### ISSUE THREAD

**Issue by gregrickaby** [WebDevStudios/wd_s#546]
If possible, please use [@wordpress/scripts](https://developer.wordpress.org/block-editor/packages/packages-scripts/) to handle builds. 

I understand there may be circumstances where we need to continue to rely on NPM scripts, however, I'd like to exhaust all possibilities of using @wordpress/scripts first.

- [x] Remove as many custom NPM script entries as possible
- [x] SVG sprites should be generated using Webpack
- [x] Icon sprites should be generated using Webpack
- [x] Images should be optimized using Webpack

**Comment by coreymcollins** [comment]
Right now we're using `imagemin` for optimization and `svgo` + `svgstore` for SVG handling. On our call this morning, we chatted about this and Ashar taking a look into some Webpack options.

**Comment by asharirfan** [comment]
I have removed the use of `imagemin` and `svgo` via NPM scripts in the theme. And added `imagemin-webpack-plugin` to handle the same workflow via @wordpress/scripts.

I wasn't able to find out a good replacement Webpack plugin for `svgstore` so I have left that script as it is for now.

I have opened a draft PR #552 in the repository. I will mark it as ready once @coreymcollins Tailwind PR is finalized.
